tests/docker_service.py

```diff
@@ -47,19 +47,15 @@ async def wait_until_responsive(
 
 
 class DockerServiceRegistry:
-    def __init__(self, use_legacy_compose: bool = False) -> None:
-        self._use_legacy_compose = use_legacy_compose
-        if os.environ.get("DOCKER_USE_LEGACY_COMPOSE") == "true" or os.environ.get("GITHUB_ACTIONS") == "true":
-            self._use_legacy_compose = True
+    def __init__(self) -> None:
         self._running_services: set[str] = set()
         self.docker_ip = self._get_docker_ip()
-        self._base_command = ["docker-compose"] if self._use_legacy_compose else ["docker", "compose"]
-        self._base_command.extend(
```

Comment:
Can we add this hook back in, but remove `GITHUB_ACTIONS` from the conditional?  This will still allow me to override it if necessary.